custom_resources/ssm.py

```diff
@@ -48,3 +48,31 @@ def name(cls):
         """
         # Keep legacy non-structured name for backward compatibility
         return ['SsmParameter']
+
+
+class ParseDict(LambdaBackedCustomResource):
+    props = {
+        'Names': ([string_types], True),  # The parameter paths including namespace
```

Comment:
Is the aws naming not rather 'Keys' ?

Reply:
`GetParameters` also refers to `Names` (https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_GetParameters.html#systemsmanager-GetParameters-request-Names), or did you mean something else?

Reply:
Ok... het is een beetje verwarrend. Soms Name, soms Key... nm then.